src/cli.js

```diff
@@ -62,12 +62,10 @@ function shx(argv) {
     if (shell.error() || (ret && ret.code && ret.code !== 0)) failed = true;
   }
 
-  if (failed) {
-    return EXIT_CODES.CMD_FAILED;
-  } else {
-    return EXIT_CODES.SUCCESS;
-  }
+  return failed ? EXIT_CODES.CMD_FAILED : EXIT_CODES.SUCCESS;
 }
 
+process.exit(shx(process.argv));
```

Comment:
Makes sense, I'll separate the functions into individual files and put this call/exit in the cli.js.